domains/akbar.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "akbarrdev",
+        "email": "mhdakbrprdana2@gmail.com"
+    },
+    "record": {
+        "CNAME": "portfolio-7to.pages.dev"
```

Comment:
Uh this seems to just be a next.js template from Vercel resulting in this not being complete